content/en/users/cloud-compute/ec3/apps/ecas/_index.md

```diff
@@ -57,57 +57,57 @@ To configure and deploy a Virtual Elastic Cluster using EC3, access the
 [EC3 platform front page](https://servproject.i3m.upv.es/ec3-ltos/index.php) and
 click on the **\"Deploy your cluster\"** link as shown in the figure below:
 
-![EC3 front page.](ecas-front.png)
+![ecas-front](ecas-front.png)
```

Comment:
Here ecas-front will be used as an alternate text, it would be better to provide something more meaningful. Same for the other image entries.

Reply:
Done!